### PR TITLE
Validate form

### PR DESCRIPTION
**Pull request by vickymarz**
### Email form validation
_In this milestone;_

- We validated the form email input using javascript
- We added error messages to show the validation errors

### CODE REVIEW

File: js/validate.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+function showSuccess(input) {
+  return displayMessage(input, '', true);
+}
+
+function validateEmail(input, invalidMsg) {
+  const email = input.value.trim();
+  if (email !== email.toLowerCase()) {
+    return showError(input, invalidMsg);
```
Comment (micheaol) [inline js/validate.js:21]: Please ensure that the error message should show up close to the `submit button`. And that the error message fits the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, margins).

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @vickymarz , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: js/validate.js (hunk 1 above)
Comment (victorokoroji) [reply js/validate.js:21]: Thank you for the feedback, but I can't seem to find on Figma how the error message should be displayed such as the color,  position, typo . Pls can you give me more clarity?

Comment (micheaol) [reply js/validate.js:21]: Please ensure the error message is close to the submit button. It's currently at the bottom of the page, I have to scroll before I can see it. Please make it close to the submit button according to the requirement.

Comment (victorokoroji) [reply js/validate.js:21]: noted. Thanks

**Review by micheaol — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hi @vickymarz , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**